tools/x3gen.py

```diff
@@ -234,8 +229,6 @@ def create(self):
 
 if __name__ == '__main__':
     print('Creating a sample PDF/X3 document.')
-    print('Due to license restrictions we can not commit the ICC profiles or images embedding them in the Git repo.')
-    print('Thus you need to provide them yourself.')
```

Comment:
The sample image is still not in the repo so this should print a warning about that.

Reply:
I'd rather add the simple image if possible. Are you allowed to share the file?

Reply:
I did draw it myself. I'd need to check whether the color profile I used for it is actually one of the redistributable ones. Adding it as a test image would be useful in any case.

Reply:
It's in master now.